docs/Installation/Installation.md

```diff
@@ -6,19 +6,121 @@ See LICENSE.txt for details.
 
 \tableofcontents
 
-This page details the installation procedure for SpECTRE on personal computers
-using x86-64 processors. For configuring SpECTRE, please refer to the
-\subpage spectre_build_system page. For instructions on installing SpECTRE on
-clusters please refer to the \subpage installation_on_clusters page. For
-instructions on installing SpECTRE on Apple Silicon Macs, please refer to the
-\subpage installation_on_apple_silicon page. Refer to the
-\subpage versioning_and_releases page for information on specific versions
-to install.
+### Running containerized releases
+
+A quick way to run the code without installing anything at all is with our
+containerized releases:
+
+```
+docker run sxscollaboration/spectre --help
+```
+
+You can also use [Apptainer/Singularity](https://apptainer.org) instead of
+Docker, which works better on computing clusters and is more convenient because
+it shares the host's file system:
+
+```
+apptainer run docker://sxscollaboration/spectre --help
+```
+
+For example, you can generate initial data for a simulation of merging black
+holes and plot the result like this:
+
+```
+apptainer run docker://sxscollaboration/spectre bbh generate-id \
+  -q 1 --chi-A 0 0 0 --chi-B 0 0 0 -D 16 -w 0.015 -a 0 -o ./bbh_id
```

Comment:
[optional] I'd advise against putting these examples in since we may change arguments/names/anything at some point, and then forget to update this, and then the very first easy thing we tell people to do won't work.

Reply:
I thought it would be nice to give an example that actually does something, but I agree with your concern. I'll revisit this when I put the initial data tutorial online and maybe just link to it.